plugins/BEdita/Core/src/Filesystem/Adapter/LocalAdapter.php

```diff
@@ -44,7 +44,7 @@ public function initialize(array $config)
     {
         $success = parent::initialize($config);
 
-        if (empty($this->_config['baseUrl']) && strpos($this->getConfig('path'), WWW_ROOT . DS) === 0) {
+        if (empty($this->_config['baseUrl']) && strpos($this->getConfig('path'), WWW_ROOT) === 0) {
             // Files are stored within the document root, so base URL can be automatically detected if not already set.
             $path = str_replace(DS, '/', substr($this->getConfig('path'), strlen(WWW_ROOT)));
```

Comment:
...instead, I'd put strlen(WWW_ROOT) + 1 here.

Reply:
also not necessary since `WWW_ROOT` contains last dir separator